### PR TITLE
added  extra points in other safety tips.

### PR DESCRIPTION
**Pull request by RAJU2529**
Just now i added extra points under other safety tips. 
I also added  some fake flash player sites list, and  how to add those sites in hosts file

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 1]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
 ## Other safety tips
 
 To further ensure that data is protected from malware as well as other threats:
+* Enable systemrestore point before installing any softwares
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
* Enable a system restore point before installing any software.
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 2]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
...
+
+* Create system image backup  in other harddisk or in usb pendrive
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
* Create a system image backup in another hard disk or USB pendrive.
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 3]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
...
+
+* Block the fake websites in hosts file which is  located in C:\Windows\System32\drivers\etc\  like which contains malwares or trojans
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
* Block the fake websites in the host file which is  located in C:\Windows\System32\drivers\etc\  and enter the sites that contain malware or Trojans 
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 4]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
 ## Other safety tips
 
 To further ensure that data is protected from malware as well as other threats:
+* Enable systemrestore point before installing any softwares
+
+* Create system image backup  in other harddisk or in usb pendrive
+
+* Block the fake websites in hosts file which is  located in C:\Windows\System32\drivers\etc\  like which contains malwares or trojans 
+by adding new line  at end , use notepad like below.If you cant able to access means , then take ownership of that file or folder
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
Use Notepad to add a new line at the end as in the following examples. If you cannot get access, take ownership of that file or folder.
```

**Comment by e0i** [comment]
@RAJU2529 Would you be able to follow-up with copy editing suggestions? Thanks.

**Comment by RAJU2529** [comment]
Yes sir. what follow up to do

**Comment by illfated** [comment]
@RAJU2529 : As the owner of this Pull Request, you need to go to every comment suggestion from JohanFreelancer9 and accept the suggested changes (based on the recommended improvements in English grammar & orthography).

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 5]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
 ## Other safety tips
 
 To further ensure that data is protected from malware as well as other threats:
+* Enable a system restore point before installing any software.
+
+* Create a system image backup in another hard disk or USB pendrive.
```
Comment (illfated) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
* Create a system image backup on another hard drive or USB pen drive.
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 6]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
...
+
+* Block the fake websites in the host file that is  located in C:\Windows\System32\drivers\etc\  and enter the sites that contain malware or Trojans.
+Use Notepad to add a new line at the end as in the following examples. If you cannot get access, take ownership of that file or folder.
+
+# flash update trojan sites
+#
+127.0.0.1	startheavilysophisticatedproduct.icu
```
Comment (illfated) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
127.0.0.1 startheavilysophisticatedproduct.icu
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 7]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
...
+127.0.0.1	startpreciseheavilyproduct.icu
```
Comment (illfated) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
127.0.0.1 startpreciseheavilyproduct.icu
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 8]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
...
+127.0.0.1	betterinstalltypeappclicks.icu
```
Comment (illfated) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
127.0.0.1 betterinstalltypeappclicks.icu
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 9]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
...
+127.0.0.1	restoremostrecentproduct.icu
```
Comment (illfated) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
127.0.0.1 restoremostrecentproduct.icu
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 10]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
...
+127.0.0.1 repaircompletelyprogressiveproduct.icu
+127.0.0.1 anyconcretetypetheclicks.icu
+127.0.0.1 startoriginalintenselyproduct.icu
+
```
Comment (illfated) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 11]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
...
 To further ensure that data is protected from malware as well as other threats:
+* Enable a system restore point before installing any software.
+
+* Create a system image backup in another hard disk or USB pendrive.
+
+* Block the fake websites in the host file that is  located in C:\Windows\System32\drivers\etc\  and enter the sites that contain malware or Trojans.
+Use Notepad to add a new line at the end as in the following examples. If you cannot get access, take ownership of that file or folder.
+
+# flash update trojan sites
+#
```
Comment (illfated) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
```

File: windows/security/threat-protection/intelligence/prevent-malware-infection.md [hunk 12]
```diff
@@ -76,6 +76,23 @@ Whenever necessary, log in as an administrator to install apps or make configura
...
 
 To further ensure that data is protected from malware as well as other threats:
+* Enable a system restore point before installing any software.
+
+* Create a system image backup in another hard disk or USB pendrive.
+
+* Block the fake websites in the host file that is  located in C:\Windows\System32\drivers\etc\  and enter the sites that contain malware or Trojans.
+Use Notepad to add a new line at the end as in the following examples. If you cannot get access, take ownership of that file or folder.
+
+# flash update trojan sites
```
Comment (illfated) [inline windows/security/threat-protection/intelligence/prevent-malware-infection.md]:
```suggestion
## Flash update trojan sites
```

**Comment by e0i** [comment]
@RAJU2529 Please follow-up with rest of the copy editing suggestions to move forward with the technical review. Thanks.

**Comment by e0i** [comment]
@RAJU2529

The PR is being closed as it has not met initial review on certain technical and editorial guidelines.

Thank you for your time and effort.

**Comment by e0i** [comment]
@officedocsbot close

**Comment by RAJU2529** [comment]
3rd PR request is closed without telling reason. then no use of adding new constitutionsbios from my side. I try to stop answering questions and feedback

**Pull request closed without merging**